src/server/routes/users.route.js

```diff
@@ -559,6 +579,34 @@ router.get('/personality', apiProtector, user.getPersonalityInfo);
 router.get('/mentorMode', apiProtector, user.getMentoringRequestStatus);
 
 /**
+ * @api {get} /users/bookmark Request bookmark list
```

Comment:
I think meaning of 'get something' is already included in HTTP method GET. when it comes to naming in uri, name with verb is not recommended.